ipwb/replay.py

```diff
@@ -525,6 +527,13 @@ def getRequestedSetting(requestedSetting):
     return Response(ipwbUtils.getIPFSAPIHostAndPort() + '/webui')
 
 
+# Lookup digest in IPFS and populate the 'message' dic using specified key.
+# The key here could either be 'header' or 'payload'.
+# Using the mutable 'message' dict instead of returning a value due to the
+# asynchronous nature of threads which is being utilized to call this function.
+def load_from_ipfs(digest, message, key):
```

Comment:
We have been using camelCaseFunctionNames whereas here you used an_underscore_delimited_function_name. Please make this consistent to match the style of the rest of the package.

Reply:
We are not consistent about camelCasing. I initially named that way, but found some functions defined with underscores, so changed it just before committing the code. Changing it back to camelCase now.